app/models/common_page.rb

```diff
@@ -0,0 +1,44 @@
+# Page model for pages with content shared by all modules
+# whilst still able to render training_module/module_item specific variables
+#
+# Pagination is disabled for these pages
+#
+class CommonPage < ContentPage
```

Comment:
Can we include `TranslateFromLocale` here and be more consistent with how we call the `I18n.t` vs `translate` - I prefer the more explicit `I18n.t` actually...

Reply:
Actually - is the `include` required?

Reply:
Thqt module is inherited, and that's because any common page requiring module specific values can get them from the module item. Like the feedback form.

Reply:
OK, minor dyslexic moment there, I thought the inheritance was the other way around.